presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcPageSource.java

```diff
@@ -74,23 +80,33 @@
     private int batchId;
     private long completedPositions;
     private boolean closed;
+    private boolean rowsDeleteLoad;
+    private Optional<BitSet> rowsDeleted = Optional.empty();
 
     public OrcPageSource(
+            OrcStorageManager orcStorageManager,
+            FileSystem fileSystem,
             Optional<ShardRewriter> shardRewriter,
             OrcBatchRecordReader recordReader,
             OrcDataSource orcDataSource,
             List<Long> columnIds,
             List<Type> columnTypes,
             List<Integer> columnIndexes,
             UUID shardUuid,
+            boolean tableSupportsDeltaDelete,
             OptionalInt bucketNumber,
-            AggregatedMemoryContext systemMemoryContext)
+            AggregatedMemoryContext systemMemoryContext,
+            Optional<UUID> deltaShardUuid)
     {
+        this.orcStorageManager = orcStorageManager;
+        this.fileSystem = fileSystem;
```

Comment:
nit: require non null for L101 and L102. But see my previous comment I don't think we need this 2 as members for `OrcPageSource`.